### PR TITLE
feat: Add vpc flow logs

### PR DESCRIPTION
**Pull request by btotharye**
Adds VPC Flow log example for python

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: python/vpc/vpc-flow-logs/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# AWS VPC with Flow Logs with Python CDK!
```
Comment (rix0rrr) [inline python/vpc/vpc-flow-logs/README.md]: Please replace this automatically-generated README with a README that describes your example.

File: python/vpc/vpc-flow-logs/app.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        # Setup IAM user for logs
+        vpc_flow_role = iam.Role(
+            self, 'FlowLog',
+            assumed_by=iam.ServicePrincipal('vpc-flow-logs.amazonaws.com')
+        )
+
+        # Create Cloudwatch log group
+        log_group = logs.LogGroup(
+            self, 'LogGroup',
+            log_group_name='vpc-flow-example',
```
Comment (rix0rrr) [inline python/vpc/vpc-flow-logs/app.py]: Hard-naming resources is a pattern that we don't want to encourage. Please let names be automatically generated.

File: python/vpc/vpc-flow-logs/app.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            retention=logs.RetentionDays('ONE_YEAR'),
```
Comment (rix0rrr) [inline python/vpc/vpc-flow-logs/app.py]: Shouldn't this be `logs.RetentionDays.ONE_YEAR` ?

File: python/vpc/vpc-flow-logs/app.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            removal_policy=core.RemovalPolicy('DESTROY')
```
Comment (rix0rrr) [inline python/vpc/vpc-flow-logs/app.py]: Same.

**Comment by btotharye** [comment]
I'll just remove this I don't really have time atm to change all this I was just trying to be helpful...

**Pull request closed without merging**